docs/guides/authoring/overview.rst

```diff
@@ -0,0 +1,328 @@
+
+For example, the Nuke node OCIOLogConvert: instead of requiring the
+user to select the appropriate log colorspace, the node performs a
+transform between ``scene_linear`` and ``compositing_log``, and the
+OCIO config specifies the project-appropriate colorspaces. This
+simplifies life for artists, as they don't have to remember which is
+the correct log colorspace for the current project - the
+OCIOLogConvert always does the correct thing.
+
+
+A typical role definition looks like this, taken from the
+:ref:`config-spivfx` example configuration:
+
+.. code-block:: yaml
+
+    roles:
+      color_picking: cpf
+      color_timing: lg10
+      compositing_log: lgf
+      data: ncf
+      default: ncf
+      matte_paint: vd8
+      reference: lnf
+      scene_linear: lnf
+      texture_paint: dt16
+
+
+All values in this example (such as ``cpf``, ``lg10`` and ``ncf``)
+refer to colorspaces defined later the config, in the ``colorspaces``
+section.
+
+
+Here is a description of the roles defined within OpenColorIO. Note
+that application developers may also define roles for config authors
+to use to control other types of tasks not listed below.
+
+Warning: applications may interpret or use these differently.
+
+* ``aces_interchange`` - defines the color space in the config that
+  implements the ACES2065-1 color space defined in SMPTE ST2065-1.
+  This role is used to convert scene-referred color spaces between
+  different configs that both define this role.
+
+* ``cie_xyz_d65_interchange`` - defines the color space in the config
+  that implements standard CIE XYZ colorimetry, adapted to a D65 white.
+  This role is used to convert display-referred color spaces between
+  different configs that both define this role.
+
+* ``color_picking`` - colors in a color-selection UI can be displayed
+  in this space, while selecting colors in a different working space
+  (e.g. ``scene_linear`` or ``texture_paint``)
+
+* ``color_timing`` - colorspace used for applying color corrections,
+  e.g. user-specified grade within an image viewer (if the application
+  uses the ``DisplayTransform::setDisplayCC`` API method)
+
+* ``compositing_log`` - a log colorspace used for certain processing
+  operations (plate resizing, pulling keys, degrain, etc). Used by the
+  OCIOLogConvert Nuke node
+
+* ``data`` - used when writing data outputs such as normals, depth
+  data, and other "non color" data. The colorspace in this role should
+  typically have ``data: true`` specified, so no color transforms are
+  applied
+
+* ``default`` - when ``strictparsing: false``, this colorspace is used
+  as a fallback. If not defined, the ``scene_linear`` role is used
+
+* ``matte_paint`` - colorspace which matte-paintings are created in
+  (for more information, :ref:`see the guide on baking ICC profiles
+  for Photoshop <userguide-bakelut-photoshop>`, and
+  :ref:`config-spivfx`)
+
+* ``reference`` - the colorspace against which the other colorspaces
```

Comment:
In a future revision, might be worth mentioning that historically some interpreted this as being a role for reference images. I agree though in being resolute about saying what it now represents, and agree on the intended meaning.

Reply:
Good point.  I added a note on reference and added a warning at the top saying the roles section needs some work.